pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -160,51 +182,69 @@ const CryptoPolicyDialog = ({
                footer={
                    <>
                        {error && <ModalError dialogError={typeof error == 'string' ? error : error.message} />}
-                       <Button id="crypto-policy-save-reboot" variant='primary' onClick={() => setPolicy(true)}>
-                           {_("Save and reboot")}
+                       {inProgress && selected === "FIPS" &&
+                           <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} className="fips-mode-progress">
+                               <Spinner isSVG size="md" />
+                               <p>{_("Applying FIPS policy... This may take a few minutes.")}</p>
```

Comment:
Can we show this for the other policies? In princile this might also take some time, no? Or would that create flickering?

Reply:
I tried to switch away from FIPS. It's much faster than enabling FIPS, but it still takes some 5s. Would be nice to make this consistent.

I.e. drop the extra `<Spinner>` (it looks really weird, see screenshot above), always says "Applying new policy...", and always do this.

Reply:
Dropping the spinner sounds fine, made the text more general as also switching policies takes a "while"